Classes/Handler/FileUploadHandler.php

```diff
@@ -194,6 +208,39 @@ public function createFileReference(InterestRequestInterface $request): Response
         );
     }
 
+    /**
+     * @param InterestRequestInterface $request
+     * @return ResponseInterface
+     */
+    public function getProductImages(InterestRequestInterface $request): ResponseInterface
```

Comment:
What are you using this method for? It is specific to pxa_product_manager and shouldn't be in this extension.